### ISSUE TITLE
[jose review] Accessibility – clickable elements

### ISSUE THREAD

**Issue by dpshelio** [thejackal360/Elena-Game-Engine#22]
clickable elements [must be focusable and should have interactive semantics](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard?utm_source=devtools&utm_medium=a11y-panel-checks-keyboard#clickable_elements_must_be_focusable_and_should_have_interactive_semantics).
